Dockerfile

```diff
@@ -1,13 +1,18 @@
-FROM python:2.7
+FROM python:2.7-slim
 
 MAINTAINER bhearsum@mozilla.com
 
-# Some versions of the python:2.7 Docker image remove this library, which
+# Some versions of the python:2.7 Docker image remove libpcre3, which
 # uwsgi needs for routing support to be enabled.
 # We may be able to remove this after https://github.com/docker-library/python/pull/137
```

Comment:
Good point - I will adjust the comment. I think I'll leave the explicit install though...it never hurts to be explicit.